PublicMapping/districtbuilder
# Limit drawing to within starting county

## Overview

- Add new checkbox in MapHeader dropdown menu, 'Limit selection to county'
- Enables end users to reduce county splits by limiting the selection of the rectangle / paintbrush tools to only select geounits from within a certain county in a single mouse drag*

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/115082262-9f219b80-9ed3-11eb-902d-f767dceb9ee6.png)


### Notes

* It _is_ possible to create a selection that spans multiple counties, if you do so in multiple distinct drags. This is the same as the [protoype](https://dbtwo.netlify.app/), however

## Testing Instructions

- Open up an editable project, and click the tool icon in the header and select "Limit selection to county"
- Select the paintbrush tool at the block or block group level, and drag an area in one county. Attempt to continue drag into another county
- Expect: Geounits in the adjacent county are not selected
- Select the rectangle tool at the block or block group level, and drag an area in one county. Attempt to continue drag into another county
- Expect: Geounits in the adjacent county are not selected


Closes #216
